packages/block-library/src/gallery/style.scss

```diff
@@ -80,26 +78,44 @@
 		}
 	}
 
-	// Responsive fallback value, 2 columns
+	// On mobile and responsive viewports, we allow only 1 or 2 columns at the most.
 	& .blocks-gallery-image,
 	& .blocks-gallery-item {
-		width: calc(100% / 2 - 16px);
+		width: calc((100% / 2) - (#{ $grid-size-large } * 2));
 	}
 
 	&.columns-1 .blocks-gallery-image,
 	&.columns-1 .blocks-gallery-item {
-		width: calc(100% / 1 - 16px);
+		width: calc(100% / 1);
```

Comment:
Is calc(100% / 1) the same as 100%?

Reply:
Yes, I think it was written this way to mimic the other column math in near context... sort of like when we use $border-width instead of 1px just to make the math more easily digestible. 

I'm fine with changing this if you like?

Reply:
Ok, I thought it was either that or some kind of CSS hack. It's probably best to change it, but it's very very minor 😄